### PR TITLE
kelda: Remove `kelda.hostIP`

### PR DESCRIPTION
**Pull request by kklin**
This commit removes the `kelda.hostIP` reference that allows containers
to reference their public IP via their environment variables. The
feature is extremely hard to implement on Kubernetes, and was only used
by the Spark blueprint.

Containers that need this feature should query their public IP at
runtime using a service such as checkip.amazon.com, or pass containers a
floating IP (or DNS name backed by a floating IP) and use placement
rules to ensure the container is scheduled at that floating IP.

### CODE REVIEW

**Comment by luise** [comment]
Can you also remove the reference to `hostIP` in the example in the docs? http://docs.kelda.io/#Container

**Comment by kklin** [comment]
I think removing it from the JSDoc is sufficient (https://github.com/kelda/kelda/pull/1379/files#diff-3f91f27ca724b47373bd64a690860984L997)

**Comment by luise** [comment]
Oh, yes, it definitely is! My bad 😀

File: minion/scheduler/scheduler.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func Run(conn db.Conn, dk docker.Client) {
 		minion := conn.MinionSelf()
 
 		if minion.Role == db.Worker {
-			runWorker(conn, dk, minion.PrivateIP, minion.PublicIP)
```
[the remark this answers is not in the document]
Comment (kklin) [reply minion/scheduler/scheduler.go]: Nope! I'll remove it.

[the remark this answers is not in the document]
Comment (kklin) [reply minion/scheduler/scheduler.go]: Do you want to take another look after I remove it?

[the remark this answers is not in the document]
Comment (luise) [reply minion/scheduler/scheduler.go]: Nah, that's fine!

**Pull request merged**